rbac/rbac/middleware.py

```diff
@@ -123,11 +119,12 @@ def _create_tenant(account):
         schema_name = create_schema_name(account)
         try:
             with transaction.atomic():
```

Comment:
From the docs, it seems that get_or_create is atomic, so maybe we don't need this? Not sure. https://docs.djangoproject.com/en/3.0/ref/models/querysets/#get-or-create

Reply:
Yea, I think you are right!